README.md

```diff
@@ -99,7 +110,11 @@ $("button").on("click", function(event){
 
 Your Answer:
 ```js
-// write code here
+var el = document.querySelectorAll("button")
+el.addEventListener("click", function(event){
+  el.style['color'] = 'red';
+})
+
```

Comment:
Unfortunately with vanilla JS you need to loop through all of the buttons and add event listeners to them individually.